packages/plugin-core/src/test/suite-integ/SeedCommand.test.ts

```diff
@@ -1,12 +1,31 @@
 import { tmpDir } from "@dendronhq/common-server";
 import { SeedService } from "@dendronhq/engine-server";
 import { TestSeedUtils } from "@dendronhq/engine-test-utils";
+import sinon from "sinon";
 import { SeedAddCommand } from "../../commands/SeedAddCommand";
 import { SeedRemoveCommand } from "../../commands/SeedRemoveCommand";
 import { DENDRON_COMMANDS } from "../../constants";
 import { expect } from "../testUtilsv2";
 import { runLegacyMultiWorkspaceTest, setupBeforeAfter } from "../testUtilsV3";
 
+function getFakedAddCommand(svc: SeedService) {
+  const cmd = new SeedAddCommand(svc);
+
+  sinon.replace(cmd, <any>"onUpdatingWorkspace", sinon.fake.resolves(null));
```

Comment:
can we return the fakes and then check that they are called? `expect( $fake.calledOnce ).toBeTruthy()`